test/quantization/core/test_quantized_tensor.py

```diff
@@ -615,7 +692,7 @@ def test_qtensor_copy(self):
             # copy from float doesn't support cuda
             device = 'cpu'
             # check copy from non-quantized to quantized
-            r = torch.randn([numel], dtype=torch.float).to(device)
+            r = torch.randn([numel], dtype=torch.float, device=device)
```

Comment:
Do you still need device to be 'cpu' here?

Reply:
yeah Its the same as the test in line 214 i.e. test_qtensor_float_assignment, the functionality for assigning a float value to an element of a quantized tensor and having it automatically inserted in a quantized form. 

Both this test and the one on 214 rely on that functionality which isn't currently supported on cuda.